playwright-e2e/tests/additional-applications.spec.ts

```diff
@@ -196,6 +195,46 @@ test.describe('Upload additional applications', () => {
       await expect(page.getByText('This is a confidential draft order and restricted viewing applies')).toBeVisible();
     });
 
+  test('CTSC uploads standard C2 application with no PBA', async ({ page,
+                                                          signInPage,
+                                                          additionalApplications,
+                                                          uploadAdditionalApplications,
+                                                          uploadAdditionalApplicationsApplicationFee,
+                                                          uploadAdditionalApplicationsSuppliedDocuments,
+                                                          submit    }) => {
+      caseName = 'CTSC standard C2 application ' + dateTime.slice(0, 10);
+      await updateCase(caseName, caseNumber, caseData);
+      await signInPage.visit();
+      await signInPage.login(CTSCUser.email, CTSCUser.password);
+      await signInPage.navigateTOCaseDetails(caseNumber);
+
+      // complete C2 application
+      await additionalApplications.gotoNextStep('Upload additional applications');
+      await uploadAdditionalApplications.checkC2Order();
+      await uploadAdditionalApplications.checkApplicationWithNotice();
+      await uploadAdditionalApplications.checkConfidentialApplicationYes();
+      await uploadAdditionalApplications.selectApplicantValue(1);
+      await uploadAdditionalApplications.clickContinue();
+
+      await uploadAdditionalApplicationsSuppliedDocuments.uploadC2Document(config.testPdfFile);
+      await page.waitForTimeout(3000); // wait for document upload
```

Comment:
timeouts are generally a bad pattern - if you can, hunt for some behaviour within the page (e.g. DOMcontentloaded or something) or a specific network request completing and build around that if possible. Same applies to line 225 and 231. 
If you struggle to find something obviously stick with the timeout but it will probably bite you at some point.